content/blog/self-hosted-search-and-deploy/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Announcing Two New Features for Self-Hosted Pulumi Cloud: Resource Search and Pulumi Deployments"
+allow_long_title: true
+date: 2024-07-02T09:08:31-07:00
+draft: false
+meta_desc: Introducing Resource Search and Pulumi Deployments in Self-Hosted Pulumi Cloud for enhanced management and deployment of cloud infrastructure.
+meta_image: meta.png
+authors:
+    - meagan-cojocar
+tags:
+    - releases
+---
+
+We are excited to introduce two powerful new features for our Self-Hosted Pulumi Cloud offering: Resource Search and Pulumi Deployments. These additions are designed to enhance the way you manage and deploy your cloud infrastructure, providing greater visibility, control, and automation.
```

Comment:
You would know better than me, but are we using Resource Search intentionally instead of Pulumi Insights?

Reply:
I use Pulumi Insights as the umbrella everything rolls up to and then feature specific names for everything else: Resource Search, Pulumi AI and Data Export.